src/schemas/ProgramDonorSummary/resolvers/summaryEntries.ts

```diff
@@ -48,14 +48,17 @@ const programDonorSummaryEntriesResolver: (
     .from(args.offset)
     .size(args.first);
 
-  const esHits: Array<{
+  type EsHits = Array<{
     _source: ElasticsearchDonorDocument;
-  }> = await esClient
+  }>;
+
+  const esHits: EsHits = await esClient
     .search({
       index: ELASTICSEARCH_PROGRAM_DONOR_DASHBOARD_INDEX,
       body: esQuery,
     })
-    .then(res => res.body.hits.hits);
+    .then(res => res.body.hits.hits)
+    .catch(err => [] as EsHits);
```

Comment:
you think we should log the error?? cause it could be something else like connection error